package.json

```diff
@@ -31,6 +31,7 @@
     "changeset": "changeset",
     "changeset:check": "changeset status --since=origin/master",
     "changeset:empty": "changeset --empty",
+    "changeset:new-version": "yarn changeset version && yarn ts-node scripts/update-config-fixtures.ts",
```

Comment:
So, this needs to be called before doing the release, right? Have you updated the release docs?

Reply:
It's a TODO